docs/guide/dev/code/index.md

```diff
@@ -39,11 +39,11 @@ This document is intended to help people become familiar with the codebase.
 
 Brooklyn is split into the following projects and sub-projects:
 
-* **``camp``**: the components for a server which speaks the CAMP REST API and which understands the CAMP YAML plan language
+* **``camp``**: the components for a server which speaks with the CAMP REST API and understands the CAMP YAML plan language
```

Comment:
Or even "speaks the CAMP REST API"

For ``utils`` I would also just say 'lower level utilities' (drop the first 2 words).

My $0.02...